packages/react-components/src/components/knowledge-graph/utils/cytoscapeParsingUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { ElementDefinition, NodeDefinition, EdgeDefinition } from 'cytoscape';
+import { NodeData, EdgeData } from '../graph/types';
+
+//Options here will be expanded to load nodes and edges with different styling, shapes ..etc
+export function getElementsDefinition(nodeData: NodeData[], edgeData?: EdgeData[]): ElementDefinition[] {
+  const elements: ElementDefinition[] = [
+    ...nodeData.map<NodeDefinition>((data) => ({
+      data: { ...data },
+    })),
+  ];
+  if (edgeData) {
+    elements.push(
+      ...edgeData.map<EdgeDefinition>((data) => ({
+        data: { lineStyle: 'solid', dashPattern: [6, 5], ...data }, //these edge line options for initial testing. Will be updated.
+      }))
+    );
+  }
+  return elements;
```

Comment:
You are right here, no need for styling stuff here and we could get rid off this helper. I removed everything related to styling and I left the helper just to construct the array , I know this could be done in the calling code but i felt i need to stress we might plot without edges just nodes. it feels more organized to have it in a helper.